src/clr-addons/progress-spinner/progress-spinner.scss

```diff
@@ -0,0 +1,16 @@
+.loading-overlay {
+  background-color: rgba(255, 255, 255, 0.8);
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  z-index: 2;
```

Comment:
I think this should be higher. if we use this inline spinner inside a modal for example the overlay would not be shown above the modal. I dont think any other element should be higher than the loading spinner so we can set an absurdly high value here, like 99999?